pkg/audit/reporter.go

```diff
@@ -12,18 +12,16 @@ import (
 	"storj.io/storj/pkg/storj"
 )
 
-// TODO(kaloyan): Make this configurable
-const maxReverifyCount = 3
-
 type reporter interface {
 	RecordAudits(ctx context.Context, req *Report) (failed *Report, err error)
 }
 
 // Reporter records audit reports in overlay and implements the reporter interface
 type Reporter struct {
-	overlay     *overlay.Cache
-	containment Containment
-	maxRetries  int
+	overlay          *overlay.Cache
+	containment      Containment
+	maxRetries       int
+	maxReverifyCount int32
```

Comment:
nit: I would define it here is `int` and typecast it to `int32` only if necessary for any expressions. This is more like a personal preference, so feel free to ignore it.